mlflow/data_mlflow_model.go

```diff
@@ -0,0 +1,45 @@
+package mlflow
+
+import (
+	"context"
+	"fmt"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/ml"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceModel() *schema.Resource {
+	return common.WorkspaceData(func(ctx context.Context, data *struct {
+		Name         string            `json:"name,omitempty" tf:"computed"`
+		Version      string            `json:"version,omitempty" tf:"computed"`
```

Comment:
These are not computed, right? Or am I missing something?

Reply:
Also, I think only Version is optional. Can you try removing `omitempty` from Name? Then you won't need the check on line 20, if I'm reading it properly.